src/utils.ts

```diff
@@ -16,3 +17,29 @@ export const createMongoNoteInstance = async (
     }).save();
   }
 };
+
+const returnUrlParameters = (url: string) => {
+  const splitUrl = url.split("/");
+  const index = splitUrl.length;
+  return [splitUrl[index - 3], splitUrl[index - 2], splitUrl[index - 1]];
+};
```

Comment:
This can go into a util file.

Reply:
Do you want us to create a utils folder containing separate utils files? This is already in our utils file...